### PR TITLE
[IMP] data_recycle: Migrate to Owl

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/data_recycle/static/src/views/data_recycle_list_view.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="DataRecycle.buttons" t-inherit="web.ListView.Buttons" t-inherit-mode="primary" owl="1">
+        <xpath expr="//t[@t-if='props.showButtons']">
+            <t t-if="nbSelected">
+                <button type="button" class="btn btn-primary o_data_recycle_validate_button" t-on-click="_onValidateClick">
+                    Validate
+                </button>
+                <button type="button" class="btn btn-secondary o_data_recycle_unselect_button" t-on-click="_onUnselectClick">
```
Comment (Williambraecky) [inline addons/data_recycle/static/src/views/data_recycle_list_view.xml:9]: `_onUnselectClick` is not defined on the controller

**Review by Williambraecky — CHANGES_REQUESTED** [review]

File: addons/data_recycle/static/src/views/data_recycle_list_view.xml (hunk 1 above)
Comment (Williambraecky) [reply addons/data_recycle/static/src/views/data_recycle_list_view.xml:9]: Nevermind

**Comment by tivisse** [comment]
@robodoo r+

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**